docs/DeveloperGuide.md

```diff
@@ -921,36 +921,242 @@ testers are expected to do more *exploratory* testing.
+   
+    4. Test case: `sedit 1 n/Alice's Chicken Rice`<br>
+       Expected: No stall is edited. Error details shown in the status message. GUI does not update.
+
+    5. Other incorrect stall edit commands to try: `sedit`, `sedit 1`, `sedit x n/Alice Chicken Rice`, `sedit x a/Chicken Street 123`(where x is larger than the list size)<br>
+       Expected: Similar to previous.
+
+### Editing a review
+
+1. Editing a review while all reviews are being shown
+
+    1. Prerequisites: List all reviews using the `rlist` command. Multiple reviews in the list.
+
+    2. Test case: `redit 1 d/20/09/2022`<br>
+       Condition: There must not exist a review with the same details (date, content and tags) as the edited review's details<br>
+       Expected: The date of the first review edited. Details of the edited review shown in the status message. GUI updates to show the edited details.
+
+    3. Test case: `redit 1 c/The chicken was dry`<br>
+       Condition: There must not exist a review with the same details (date, content and tags) as the edited review's details<br>
+       Expected: The content of the first review edited. Details of the edited review shown in the status message. GUI updates to show the edited details.
+   
+    4. Test case: `redit 1 d/2022/09/20`<br>
+       Expected: No review is edited. Error details shown in the status message. GUI does not update.
+
+    5. Other incorrect review edit commands to try: `redit`, `redit 1`, `redit x d/20/09/2022`, `redit x c/The chicken was dry`(where x is larger than the list size)<br>
+       Expected: Similar to previous.
+
+### Find stalls
+
+1. Finding stalls by name and/or tags
+
+    1. Prerequisites: At least one stall entry exists that we are finding with name and/or tag exists.
+   
+    2. Test case: `sfind n/chicken`<br>
+       Condition: There is a stall with the word `chicken` in its name<br>
+       Expected: Status message responds with the number of stalls that matches the name in command. GUI updates to show stalls that matches the name in command. 
+   
+    3. Test case: `sfind t/opendaily`<br>
+       Condition: There is a stall with the tag `opendaily`<br>
+       Expected: Status message responds with the number of stalls that matches the tag in command. GUI updates to show stalls that matches the tag in command.
+   
+    4. Test case: `sfind n/chicken t/opendaily`<br>
+       Condition: There is a stall with the word `chicken` in its name and with the tag `opendaily`<br>
+       Expected: Status message responds with the number of stalls that matches the name and tag in command. GUI updates to show stalls that matches the name and tag in command.
+   
+    5. Test case: `sfind n/Alice's`<br>
+       Expected: No stalls are found. Error details shown in the status message. GUI does not update.
+
+    6. Other incorrect stall find commands to try: `sfind`, `sfind n/`, `sfind a/Ang Mo Kio`<br>
```

Comment:
sfind n/ is accepted, finds 0 stalls as there are no keywords.

Reply:
Fixed.